modules/Core/includes/endpoints/ListUsersEndpoint.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * No params
+ * 
+ * @return string JSON Array
+ */
+class ListUsersEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'listUsers';
+        $this->_module = 'Core';
+        $this->_description = 'List all users on the NamelessMC site';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.